applications/FluidDynamicsHydraulicsApplication/tests/test_hydraulic_fluid_auxiliary_utilities.py

```diff
@@ -0,0 +1,88 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.FluidDynamicsHydraulicsApplication as KratosFluidHydraulics
+import KratosMultiphysics.KratosUnittest as UnitTest
+import KratosMultiphysics.kratos_utilities as KratosUtils
+
+class HydraulicFluidAuxiliaryUtilitiesTest(UnitTest.TestCase):
+
+    def setUp(self):
+        """Generates the model part to work on."""
+
+        # Set up the domain model parts
+        self.model = Kratos.Model()
+        test_model_part = self.model.CreateModelPart("TestModelPart")
+        inlet_model_part = test_model_part.CreateSubModelPart("Inlet")
+        test_model_part.ProcessInfo.SetValue(Kratos.DOMAIN_SIZE, 3)
+
+        # Add VELOCITY variable to be used as DOF and DISTANCE
+        test_model_part.AddNodalSolutionStepVariable(Kratos.VELOCITY)
+        test_model_part.AddNodalSolutionStepVariable(Kratos.DISTANCE)
+
+        # Modelpart: The inlet condition
+        # 7 - 8 - 9
+        # | \ | \ |
+        # 4 - 5 - 6
+        # | \ | \ |
+        # 1 - 2 - 3
+
+        # Populate model part nodes
+        test_model_part.CreateNewNode(1, 0.0, 0.0, 0.0)
+        test_model_part.CreateNewNode(2, 0.0, 1.0, 0.0)
+        test_model_part.CreateNewNode(3, 0.0, 2.0, 0.0)
+        test_model_part.CreateNewNode(4, 0.0, 0.0, 1.0)
+        test_model_part.CreateNewNode(5, 0.0, 1.0, 1.0)
+        test_model_part.CreateNewNode(6, 0.0, 2.0, 1.0)
+        test_model_part.CreateNewNode(7, 0.0, 0.0, 2.0)
+        test_model_part.CreateNewNode(8, 0.0, 1.0, 2.0)
+        test_model_part.CreateNewNode(9, 0.0, 2.0, 2.0)
+        inlet_model_part.AddNodes([1, 2, 3, 4, 5, 6, 7, 8, 9])
+
+        # Create inlet conditions
+        prop_0 = test_model_part.Properties[0]
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 1, [1, 2, 4], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 2, [2, 5, 4], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 3, [2, 3, 5], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 4, [3, 6, 5], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 5, [4, 5, 7], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 6, [5, 8, 7], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 7, [5, 6, 8], prop_0)
+        inlet_model_part.CreateNewCondition("SurfaceCondition3D3N", 8, [6, 9, 8], prop_0)
+
+        # Add velocity DOFs
+        for node in test_model_part.Nodes:
+            node.AddDof(Kratos.VELOCITY_X)
+            node.AddDof(Kratos.VELOCITY_Y)
+            node.AddDof(Kratos.VELOCITY_Z)
+            node.AddDof(Kratos.DISTANCE)
+
```

Comment:
```suggestion
        # Add velocity DOFs
        for node in test_model_part.Nodes:
            node.AddDof(Kratos.VELOCITY_X)
            node.AddDof(Kratos.VELOCITY_Y)
            node.AddDof(Kratos.VELOCITY_Z)
            node.AddDof(Kratos.DISTANCE)

```
You shouldn't need to have DOFs for the tests below.

Reply:
Yes, you are right, I will delete them